resources/views/site/index.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Html\Html;
+
```

Comment:
`$layoutParameters` should be declared via phpdoc.

Reply:
done.